new-log-viewer/src/services/LogFileManager.ts

```diff
@@ -99,17 +112,19 @@ class LogFileManager {
      * File object.
      * @param pageSize Page size for setting up pagination.
      * @param decoderOptions Initial decoder options.
+     * @param chunkResultsHandler
```

Comment:
For consistency, shall we rename it as `onSearchResults` or `onQueryResults`?

Reply:
onQueryResults

Reply:
but for consistency, we should also edit the chunkResultsHandler in MainWorker.ts right